File: solutions/jose-rivas/csharp/p-04-loops/exercise-01.cs
using System;

class Program
{
    static void Main()
    {
        try
        {
            Console.Write("Entrez un nombre N : ");
            int N = int.Parse(Console.ReadLine());

            if (N < 1)
            {
                Console.WriteLine("Erreur : N doit etre >= 1");
                return;
            }

            int somme = 0;

            for (int i = 1; i <= N; i++)
            {
                somme = somme + i;
            }

            Console.WriteLine("La somme de 1 a " + N + " est : " + somme);
        }
        catch
        {
            Console.WriteLine("Erreur : veuillez entrer un nombre entier.");
        }
    }
}

File: solutions/jose-rivas/python/p-04-loops/exercise-01.py
try:
    N = int(input("Entrez un nombre N : "))

    if N < 1:
        print("Erreur : N doit etre >= 1")
    else:
        somme = 0

        for i in range(1, N + 1):
            somme = somme + i

        print("La somme de 1 a", N, "est :", somme)

except:
    print("Erreur : Veuillez entrer un nombre valide.")